src/arm64/executor/main.c

```diff
@@ -361,9 +363,17 @@ static ssize_t enable_pre_run_flush_store(struct kobject *kobj, struct kobj_attr
 static ssize_t measurement_mode_store(struct kobject *kobj, struct kobj_attribute *attr,
                                       const char *buf, size_t count)
 {
-    if (buf[0] == 'P')
+    if (buf[0] == 'F')
+    {
+        measurement_template = (char *)&template_l1d_flush_reload;
+    }
+    else if (buf[0] == 'P')
+    {
+        measurement_template = (char *)&template_l1d_prime_probe;
+    }
+    else
     {
-        measurement_template = (char *)&template_l1d_flush_reload;  // template_l1d_prime_probe is an alternative but is currently non-functional
+        printk(KERN_ERR "arm64_executor: Neither l1d_flush_reload nor l1d_prime_probe is enabled.\n");
```

Comment:
Return error code (-1) to make the error more explicit

Reply:
Good idea. Return code added!